HISTORY.rst

```diff
@@ -11,7 +11,17 @@ Incompatible changes:
 * Removed ``aldjemy.core.Cache.sa_models``.
 * Removed ``aldjemy.core.Cache.meta``.
 * Removed ``aldjemy.orm.prepare_models``.
+* Removed ``aldjemy.table.get_all_django_models``.
+* Merged ``aldjemy.types`` and ``aldjemy.postgres`` into ``aldjemy.table``.
```

Comment:
Note that this broke our settings using django-configurations, because importing `aldjemy.table` requires settings to be configured, while it was not necessary with importing `aldjemy.types`.
`django.core.exceptions.ImproperlyConfigured` gets triggered via https://github.com/aldjemy/aldjemy/blob/aff2d2409fd824f499a5feacc7f2023e646f645f/aldjemy/table.py#L74.

There's a workaround for this (using django-configurations `setup` method to delay it), but thought you wanted to know, and I also think it might be worth fixing/improving - so feel free to create an issue from this comment.

Reply:
Oh, that totally makes sense @blueyed, and it probably affect my work project as well. It definitely needs to be fixed. I'll open an issue and request your thoughts on the best way to do fix it there. Out of curiosity, what types do you need to add? Might they be possible to add to the default configuration?

Reply:
We use `ALDJEMY_DATA_TYPES = {'DateRangeField': simple(DATERANGE)}`.

Reply:
That sounds like a great addition. #169 tracks that.